app/src/main/kotlin/com/adesso/movee/base/BaseFragmentDecorator.kt

```diff
@@ -69,6 +43,8 @@ class BaseFragmentDecorator<VM : BaseAndroidViewModel, B : ViewDataBinding>(
                 handleNavigation(command)
             }
         }
+        // fragment.viewLifecycleOwner.observeEvent(viewModel.navigation, ::handleNavigation)
+        // viewModel.navigation.observeEvent(fragment.viewLifecycleOwner, ::handleNavigation)
```

Comment:
This code commented out, `observeEvent` extension is unused in this case.

Reply:
done